src/mesido/esdl/esdl_heat_model.py

```diff
@@ -2290,7 +2290,7 @@ def convert_gas_boiler(self, asset: Asset) -> Tuple[GasBoiler, MODIFIERS]:
         for port in asset.in_ports:
             if isinstance(port.carrier, esdl.GasCommodity):
                 density = get_density(asset.name, port.carrier)
-                internal_energy = get_internal_energy(asset.name, port.carrier)
+                internal_energy = get_energy_content(asset.name, port.carrier)
```

Comment:
shouldn't we maybe use a different word instead of internal_energy here? it gets confusion later, when you access asset.internal_energy (at the assets when creating constraints), one might think youre getting the internal energy instead of energy_content

Reply:
Good point, thank. I have changes self.internal energy to self.energy_content